terraform/critical_staging/main.tf

```diff
@@ -1,16 +1,23 @@
 module "critical" {
   source = "../modules/critical"
 
-  namespace = "${local.namespace}-staging"
```

Comment:
Err… are you sure you want to do this? Won't this cause all the bucket/database names to collide?